torch/distributed/rpc/api.py

```diff
@@ -8,13 +8,27 @@
 from .internal import _internal_rpc_pickler, PythonUDF
 
 import datetime
+import contextlib
 import functools
 import sys
 import torch
 
 
 _agent = None
 
+_default_pickler = _internal_rpc_pickler
+
+@contextlib.contextmanager
+def _UseRPCPickler(rpc_pickler):
+    r"""
+    rpc_pickler: (.internal._InternalRPCPickler) Overrides the default RPC pickler
```

Comment:
You mean `_internal_rpc_pickler`?

Reply:
.internal._InternalRPCPickler is the type